en/backup-to-s3.md

```diff
@@ -157,15 +173,16 @@ Refer to [Ad-hoc full backup prerequisites](backup-to-aws-s3-using-br.md#prerequ
         secretName: backup-demo1-tidb-secret
     s3:
         provider: aws
-        # region: us-east-1
+        region: ${region}
+        bucket: ${bucket}
         # storageClass: STANDARD_IA
         # acl: private
         # endpoint:
     storageClassName: local-storage
     storageSize: 10Gi
     ```
 
-In the above two examples, all data of the TiDB cluster is exported and backed up to Amazon S3 and Ceph respectively. You can ignore the `region`, `acl`, `endpoint`, and `storageClass` configuration items in the Amazon S3 configuration. S3-compatible storage types other than Amazon S3 can also use configuration similar to that of Amazon S3. You can also leave the configuration item fields empty if you do not need to configure these items as shown in the above Ceph configuration.
```

Comment:
I think ceph only need to fill endpoint

Reply:
Ceph also supports acl, storageClass, etc.